packages/pds/src/context.ts

```diff
@@ -162,6 +168,16 @@ export class AppContext {
       : undefined
 
     const appViewAgent = new AtpAgent({ service: cfg.bskyAppView.url })
+
+    const alternateAppviewAgent = cfg.bskyAppView.alternate
+      ? new AtpAgent({ service: cfg.bskyAppView.alternate.url })
+      : undefined
+    const alternateAppviewUsers = (
+      cfg.bskyAppView.alternate?.users ?? []
+    ).reduce((acc, cur) => {
+      return acc.set(cur, true)
+    }, new Map<string, boolean>())
```

Comment:
Really doesn't make a practical difference but a set would be quite tidy here!
```ts
const alternateAppviewUsers = new Set(cfg.bskyAppView.alternate?.users)
```

Reply:
oh doh